src/shared/schema/us/types/fields.ts

```diff
@@ -65,6 +66,40 @@ export interface EntryNavigationFields {
     workbookTitle?: string | null;
 }
 
+export interface SharedEntryPermissions {
```

Comment:
Here and further, I suggest removing unused fields from the inetfrase. They can be added later when they are useful in the UI code, otherwise for now they just make it difficult to read the code.